micro-rdk-installer/src/nvs/metadata.rs

```diff
@@ -53,16 +59,16 @@ fn get_nvs_metadata_from_entry(line: &[u8; 30]) -> Result<Option<NVSMetadata>, E
                 unreachable!()
             }
         });
-        let size = u32::from_le_bytes(match line[6..10].try_into(){
+        let size = u32::from_le_bytes(match line[6..10].try_into() {
             Ok(val) => val,
             Err(_) => {
                 unreachable!()
             }
         });
-        return Ok(Some(NVSMetadata{
+        return Ok(Some(NVSMetadata {
```

Comment:
don't want to be pedantic but can we remove the spaces l68 & 62.
Also do we run cargo fmt for the micro-rd-installer ?

Reply:
we don't, I'll add it to the workflow. Also, cargo fmt added these spaces, should I still try to leave them out?